iggy/src/identifier.rs

```diff
@@ -8,21 +8,32 @@ use serde_with::serde_as;
 use std::fmt::Display;
 use std::str::FromStr;
 
+/// `Identifier` represents the unique identifier of the resources such as stream, topic, partition, user etc.
+/// It consists of the following fields:
+/// - `kind`: the kind of the identifier.
+/// - `length`: the length of the identifier payload.
+/// - `value`: the binary value of the identifier payload.
 #[serde_as]
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 pub struct Identifier {
+    /// The kind of the identifier.
     pub kind: IdKind,
+    /// The length of the identifier payload.
```

Comment:
Maybe it's worth including information about [min,max] length of Identifier ?

Reply:
Added max length.